dojo/api.py

```diff
@@ -828,10 +828,10 @@ def validation(self):
 
 
 """
-    /api/v1/jira_configurations/
+    /api/v1/JIRA_Configurations/
```

Comment:
Please, do not mix naming conventions. I think the original you changed was way better.
This makes me think of `is_Mitigated`. Please don't do this :-)

I know it's already in plenty of places, but let's not make it worse. And I am guessing you actually did this to match the rest.

Only 1 comment, but applies in several places.

Reply:
it's just a comment